backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/ui/web/order/BOMExploderCommand.java

```diff
@@ -113,7 +122,7 @@ public BOMExploderCommand(@Nullable final BOMUse bomToUse,
 			}
 			final ProductBOMLineId bomLineId = ProductBOMLineId.ofRepoId(bomLine.getPP_Product_BOMLine_ID());
 			final ProductId bomLineProductId = ProductId.ofRepoId(bomLine.getM_Product_ID());
-			final BigDecimal bomLineQty = bomsService.computeQtyRequired(bomLine, bomProductId, initialCandidate.getQty());
+			final BigDecimal bomLineQty = bomsService.computeQtyRequired(bomLine, bomProductId, initialCandidate.getQty().toBigDecimal());
```

Comment:
i think u shall keep it as Quantity and use toBigDecimal() only in place where is needed.
More, 
* the `bomUomId` from the line below i think shall be the UOM of this qty
* on line 142  are construction again this qty: `.qty(Quantitys.create(bomLineQty, bomUomId))`

Reply:
makes sense.